### PR TITLE
feat: use dns-discovery instead of predefined list

### PR DESCRIPTION
**Pull request by danisharora099**
## This PR
Uses DNS Discovery to find nodes instead of providing a predefined list of nodes

## Notes
Declares a module for `@waku/dns-discovery` (ref: https://github.com/waku-org/js-waku/issues/1124) -- will be followed up by another PR -- tracking here: https://github.com/waku-org/js-waku-examples/issues/191

### CODE REVIEW

**Review by weboko — APPROVED** [review]

File: examples/web-chat/src/App.tsx [hunk 1]
```diff
@@ -200,10 +195,17 @@ export default function App() {
...
 async function initWaku(setter: (waku: WakuLight) => void) {
   try {
+    const publicKey = "AOGECG2SPND25EEFMAJ5WF3KSGJNSGV356DSTL2YVLLZWIV6SAYBM";
+    const fqdn = "test.waku.nodes.status.im";
+    const enrTree = `enrtree://${publicKey}@${fqdn}`;
     const waku = await createLightNode({
       libp2p: {
         peerDiscovery: [
-          bootstrap({ list: getPredefinedBootstrapNodes(selectFleetEnv()) }),
+          wakuDnsDiscovery(enrTree, {
```
Comment (fryorcraken) [inline examples/web-chat/src/App.tsx:204]: There should not be 3 variable. Even 1 variable could be avoided:

```
wakuDnsDiscovery("enrtree://AOGECG2SPND25EEFMAJ5WF3KSGJNSGV356DSTL2YVLLZWIV6SAYBM@test.waku.nodes.status.im", ...
```

File: examples/web-chat/.cspell.json [hunk 2]
```diff
@@ -3,6 +3,7 @@
   "$schema": "https://raw.githubusercontent.com/streetsidesoftware/cspell/master/cspell.schema.json",
   "language": "en",
   "words": [
+    "AOGECG2SPND25EEFMAJ5WF3KSGJNSGV356DSTL2YVLLZWIV6SAYBM",
```
Comment (fryorcraken) [inline examples/web-chat/.cspell.json:6]: just add `enrtree://` in the patterns to ignore.

File: examples/web-chat/src/App.tsx [hunk 3]
```diff
@@ -200,10 +195,17 @@ export default function App() {
...
+            store: 1,
+            filter: 2,
+            lightpush: 2,
```
Comment (fryorcraken) [inline examples/web-chat/src/App.tsx:207]: let's do 1 for all for now, until we dog food peer exchange

File: examples/web-chat/.cspell.json (hunk 2 above)
Comment (yakimant) [reply examples/web-chat/.cspell.json:6]: https://github.com/waku-org/js-waku-examples/pull/278

**Pull request merged**